paasta_tools/cli/cmds/status.py

```diff
@@ -1124,6 +1132,112 @@ def print_flink_status(
+            return PaastaColors.red("Stopping")
+    elif status.desired_state == "start":
+        if num_replicasets == 0:
+            return PaastaColors.yellow("Starting")
+        if num_replicasets == 1:
+            if num_ready_replicas < status.desired_instances:
+                return PaastaColors.yellow("Launching replicas")
+            else:
+                return PaastaColors.green("Running")
+        else:
+            replicasets = sorted(status.replicasets, key=lambda x: x.create_timestamp)
+            git_shas = {r.git_sha for r in replicasets}
+            config_shas = {r.config_sha for r in replicasets}
+            bouncing_to = []
+            if len(git_shas) > 1:
+                bouncing_to.append(replicasets[0].git_sha[:8])
+            if len(config_shas) > 1:
+                bouncing_to.append(replicasets[0].config_sha)
+
+            bouncing_to_str = ", ".join(bouncing_to)
+            return PaastaColors.yellow(f"Bouncing to {bouncing_to_str}")
+    else:
+        return PaastaColors.red("Unknown")
+
+
+def get_versions_table(replicasets) -> List[str]:
+    if len(replicasets) == 0:
+        return [PaastaColors.red("There are no running versions for this instance")]
+    elif len(replicasets) == 1:
+        return get_version_table_entry(replicasets[0])
+    else:
+        replicasets = sorted(replicasets, key=lambda x: x.create_timestamp)
+        config_shas = {r.config_sha for r in replicasets}
+        if len(config_shas) > 1:
+            show_config_sha = True
+        else:
+            show_config_sha = False
+
+        table: List[str] = []
+        table.extend(
+            get_version_table_entry(
+                replicasets[0],
+                version_name_suffix="new",
+                show_config_sha=show_config_sha,
+            )
+        )
+        for replicaset in replicasets[1:]:
+            table.extend(
+                get_version_table_entry(
+                    replicaset,
+                    version_name_suffix="old",
+                    show_config_sha=show_config_sha,
+                )
+            )
+        return table
+
+
+def get_version_table_entry(
+    replicaset, version_name_suffix: str = None, show_config_sha: bool = False,
+) -> List[str]:
+    version_name = replicaset.git_sha[:8]
+    if show_config_sha:
+        version_name += f", {replicaset.config_sha}"
+    if version_name_suffix is not None:
+        version_name += f" ({version_name_suffix})"
+    version_name = PaastaColors.blue(version_name)
+
+    start_datetime = datetime.fromtimestamp(replicaset.create_timestamp)
+    humanized_start_time = humanize.naturaltime(start_datetime)
+    entry = [f"{version_name} - Started {start_datetime} ({humanized_start_time})"]
+    replica_states = get_replica_states(replicaset.pods)
+    entry.append(f"  Replica States: {replica_states}")
+    return entry
+
+
+def get_replica_states(pods) -> str:
+    return PaastaColors.green(f"{len(pods)} Running")
```

Comment:
nit: Add a todo comment here? It is slightly confusing to see an always green value, when we're really going to be adding many more.